xbmc/video/VideoInfoScanner.h

```diff
@@ -26,12 +26,13 @@ namespace VIDEO
 
   typedef struct SScanSettings
   {
-    SScanSettings() { parent_name = parent_name_root = noupdate = exclude = false; recurse = 1;}
+    SScanSettings() { parent_name = parent_name_root = noupdate = exclude = all_ext_audio = false; recurse = 1;}
     bool parent_name;       /* use the parent dirname as name of lookup */
     bool parent_name_root;  /* use the name of directory where scan started as name for files in that dir */
     int  recurse;           /* recurse into sub folders (indicate levels) */
     bool noupdate;          /* exclude from update library function */
     bool exclude;           /* exclude this path from scraping */
+    bool all_ext_audio;     /* treat all audio files in video directory as external tracks */
```

Comment:
`bool m_allExtAudio; `

As this is a member variable and variable name should follow the coding guidelines.